client/components/StarRating.jsx

```diff
@@ -27,7 +27,7 @@ var StarRatings = ({isReadOnly,avgRatings,starName}) => {
     return arr;
   }
   return (
-    <Wrapper>
+    <Wrapper starSize={starSize} isReadOnly={isReadOnly}>
       {createRadioButton()}
```

Comment:
Curious, why is createRadioButton a function that you execute and return the array of react fragments instead of just creating that array and using it directly?

Reply:
Yeah. I could do that. Having it in a function helps me to keep my code scalable and clean. I can pass arguments in the future to the function and create as many star i want.